api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanging.md

```diff
@@ -5,28 +5,28 @@ default: null
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed before selection changes.
 
 ##### param(e): Object
-<!-- Description goes here -->
+Information about the event.
 
 ##### field(e.addedItems): Array<any>
-<!-- Description goes here -->
+The data of the items that have been selected.
 
 ##### field(e.cancel): Boolean | Promise<Boolean>
-<!-- Description goes here -->
+Allows you to cancel selection. 
 
 ##### field(e.component): {WidgetName}
-<!-- Description goes here -->
+The UI component's instance.
 
 ##### field(e.element): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "UI component" }
 
 ##### field(e.removedItems): Array<any>
-<!-- Description goes here -->
+The data of the items whose selection has been canceled.
```

Comment:
"items whose selection has been canceled" - this phrase is a little complicated. You wouldn't say "I canceled this item's selection". You would say "I deselected this item" or "I removed this item from selection". 

```suggestion
The data of the items removed from selection.
```

or 

```suggestion
The data of the items that have been deselected.
```

P.S. I use the first variant in all other edits. But if you like the second better, you can use it instead.